netty/src/main/java/io/grpc/transport/netty/NettyClientTransport.java

```diff
@@ -205,12 +218,12 @@ private void notifyShutdown(Throwable t) {
       }
     }
     if (notifyShutdown) {
-      listener.transportShutdown(s);
+      listener.transportShutdown(status);
```

Comment:
A comment on line 217 (not code from this PR) ... do we need the `if (!shutdown)` check at all? I think it would be more readable to just always set `shutdown = true`.  WDYT?